pycloudlib/azure/instance.py

```diff
@@ -0,0 +1,111 @@
+# This file is part of pycloudlib. See LICENSE file for license information.
+"""Azure instance."""
+
+from pycloudlib.instance import BaseInstance
+
+
+class AzureInstance(BaseInstance):
+    """Azure backed instance."""
+
+    _type = 'azure'
+
+    def __init__(self, key_pair, client, instance):
+        """Set up instance.
+
+        Args:
+            key_pair: SSH key object
+            client: Azure compute management client
+            instance: created azure instance object
+        """
+        super(AzureInstance, self).__init__(key_pair)
+
+        self._client = client
+        self._instance = instance
+        self.boot_timeout = 300
+        self.status = "active"
+
+    def wait_for_delete(self):
+        """Wait for instance to be deleted."""
+        raise NotImplementedError
+
+    def wait_for_stop(self):
+        """Wait for instance stop."""
+        raise NotImplementedError
```

Comment:
Also let's drop these.

Reply:
If we remove those methods we will get some pylint warnings, since they are abstract methods in the base class.